.travis.yml

```
@@ -36,10 +37,19 @@ install:
           -DopenPMD_USE_PYTHON=OFF -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF
           -DCMAKE_INSTALL_PREFIX=/usr/local;
       fi
+    - if [ "${WARPX_CI_PSATD_RZ:-FALSE}" == "TRUE" ]; then
+        sudo cmake-easyinstall git+https://bitbucket.org/icl/blaspp.git
+          -DBLAS_LIBRARY=generic -DUSE_OPENMP=off -DBLASPP_BUILD_TESTS=off -DCMAKE_INSTALL_PREFIX=/usr/local;
+        sudo cmake-easyinstall git+https://bitbucket.org/icl/lapackpp.git
+          -DUSE_OPENMP=off -DLAPACKPP_BUILD_TESTS=off -DCMAKE_INSTALL_PREFIX=/usr/local;
```

Totally minimal upgrade:
If you want to go super-fancy and save a few characters, I recently added these updates:
```suggestion
        sudo cmake-easyinstall --prefix=/usr/local git+https://bitbucket.org/icl/blaspp.git
          -DBLAS_LIBRARY=generic -DUSE_OPENMP=off -DBLASPP_BUILD_TESTS=off;
        sudo cmake-easyinstall --prefix=/usr/local git+https://bitbucket.org/icl/lapackpp.git
          -DUSE_OPENMP=off -DLAPACKPP_BUILD_TESTS=off;
```

If you add after the `sudo curl -L` install line this:
```
- export CEI_SUDO="sudo"
```
then we can even replace the `sudo cmake-easyinstall` calls with `cmake-easyinstall` (and `sudo` will only be used for install, not for download and build).

---

This looks good. In your suggested change, you did remove the 'sudo' but didn't add the CEI_SUDO. Was that intended? Adding CEI_SUDO should happen in a different PR since it would affect WARPX_CI_OPENPMD also.

---

Feel free to change it for openPMD in this PR as well. I could not suggest the lines in openPMD as the gui did not allow me to :) But I'll push a commit on your PR directly :)

---

Thanks for making the changes!